suite/tests/runmulti.cmake

```diff
@@ -118,7 +118,6 @@ endmacro()
 process_cmdline(precmd ON ignore)
 
 process_cmdline(cmd OFF tomatch)
-message("output: |${tomatch}|")
```

Comment:
During the review of the previous PR, I thought this was intended.  But I see now that there's another message down below that prints it only for mismatch cases.

Reply:
Right, usually I mark temporary output with a comment so it's easy to find later but failed to do so for this case.